src/error.cpp

```diff
@@ -2,6 +2,12 @@
 #include <string>
 
 using Exception = RadProto::Exception;
+Exception::Exception(const boost::system::error_code& errorCode, std::string message)
+    : std::runtime_error(errorCode.message() + " - " + message),
```

Comment:
I guess, the meaning of this constructor is to completely substitute the autogenerated message. So not `errorCode.message() + " - " + message` but just `message`.